test/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bedrock-web-vc-store-test",
+  "version": "0.0.1-0",
+  "private": true,
+  "scripts": {
+    "test": "npm run test-node",
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require test-mocha.js 10-api.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report",
+    "lint": "eslint ."
```

Comment:
Is the lint script at the top level enough?  Could remove this script and the eslint deps from test dir.